CHANGELOG.md

```diff
@@ -1,39 +0,0 @@
-# CHANGELOG
```

Comment:
I'd keep this and either update the `greensidetech` or leave it (it should redirect)